### PR TITLE
Add example of trait with function call.

### PR DESCRIPTION
**Pull request by tomasnorre**
This adds an example of how to call a function on a trait that is added with `uses()`

### CODE REVIEW

**Comment by octoper** [comment]
cc @owenvoke

File: underlying-test-case.md [hunk 1]
```diff
@@ -79,6 +79,14 @@ use Illuminate\Foundation\Testing\RefreshDatabase;
 uses(TestCase::class, RefreshDatabase::class)->in(__DIR__);
 ```
 
+In some cases it is not enough to just add the trait with `uses()` you can also call a function on the trait. 
+Can be done like this: 
+```php
+uses(CreatesApplication::class);
+beforeEach()
+    ->createApplication();
+```
```
Comment (owenvoke) [inline underlying-test-case.md:88]: I'd probably put the trait method call onto the same line just to be a bit cleaner. Perhaps we could also change the wording to be more consistent with the other examples?

```suggestion
In some cases it is not enough to just add the trait with `uses()`, for these situations you can also call a function on the trait like so:

```php
uses(CreatesApplication::class);

beforeEach()->createApplication();
```
```

**Review by owenvoke — COMMENTED** [review]
Apologies, this was lost in my notifications. 😬

**Review by owenvoke — APPROVED** [review]

**Comment by owenvoke** [comment]
Thanks @tomasnorre, this is perfect. And apologies for the really long delay before merging.

**Comment by tomasnorre** [comment]
Don't worry. Just happy that I can contribute.

**Pull request merged**